modules/SofaGeneralMeshCollision/CMakeLists.txt

```diff
@@ -42,6 +39,11 @@ list(APPEND SOURCE_FILES
     ${SOFAGENERALMESHCOLLISION_SRC}/TriangleOctreeModel.cpp
     )
 
+find_package(SofaBase REQUIRED) # SofaBaseCollision
```

Comment:
SofaBase or SofaBaseCollision?

Reply:
SofaBase, because SofaBaseCollision is not packaged yet (so you cannot do find_package(SofaBaseCollision)